src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
@GrahamCampbell , @taylorotwell isn't 5 year old kid who can't speak for himself. Stop jumping into every convo mate. Reject the PR, I am sure those spaces mean more to this project than our contributions!